Resources/doc/operations.md

```diff
@@ -21,8 +21,7 @@ By default, the following operations are automatically enabled:
 ## Disabling operations
 
 If you want to disable some operations (e.g. the `DELETE` operation), you must register manually applicable operations using
-the operation factory class, `Dunglas\ApiBundle\Resource::addCollectionOperation()` and `Dunglas\ApiBundle\Resource::addCollectionOperation()`
-methods.
```

Comment:
It's `createItemOperation` and `createCollectionOperation`: https://github.com/dunglas/DunglasApiBundle/blob/master/Api/Operation/OperationFactory.php#L45-L77